specs/topdown.md

```diff
@@ -0,0 +1,84 @@
+#Topdown Finality
+
+The topdown finality propagates the states in the parent subnet to the child blockchain. The following data are passed:
+
+- Parent block height + block hash
+- Transactions from the parent to the child subnet
+- Validator changes from parent to the child subnet
+
+The finality is represented in contract in the form of:
+
+```solidity
+// Link: https://github.com/consensus-shipyard/ipc/blob/7af25c4c860f5ab828e8177927a0f8b6b7a7cc74/contracts/src/structs/CrossNet.sol#L11
+struct ParentFinality {
+    uint256 height;
+    bytes32 blockHash;
+}
+```
+
+The validator changes are represented as an array of Changes while topdown messages are an array of [IPCEnvelope](https://github.com/consensus-shipyard/ipc/blob/7af25c4c860f5ab828e8177927a0f8b6b7a7cc74/contracts/src/structs/CrossNet.sol#L63).
+
+From a high level point of view, the topdown finality works as follows:
+
+- Parent finality process will first fetch the last committed `ParentFinality`. If there is no previous committed parent finality, the genesis block is [used](https://github.com/consensus-shipyard/ipc/blob/7af25c4c860f5ab828e8177927a0f8b6b7a7cc74/fendermint/vm/topdown/src/sync/mod.rs#L36).
```

Comment:
Can you describe what and when triggers "Parent finality process"?

Reply:
There is actually no one triggering the process, it's always running in the background once enabled. I have added this into the spec.